redisdb/tests/compose/1m-2s.compose

```diff
@@ -0,0 +1,32 @@
+version: '3.5'
+
+services:
+  redis-master:
+    image: "redis:${REDIS_VERSION}"
+    ports:
+      - "6382:6382"
```

Comment:
Do we want to let docker choose an open local port to expose 6382 ? We would detect it using docker inspect or docker ps

Reply:
Albeit more elegant, this would add a fair amount of complexity, provided that those ports are used in the compose file itself (`redis-server --port 6380 --slaveof redis-master 6382
`) and inside the tests, not sure it's worth it.

Reply:
indeed